doc/source/license.rst

```diff
@@ -1,4 +1,8 @@
 License
 =======
 
+.. highlight:: none
+
 .. literalinclude:: ../../LICENSE
+
+.. highlight:: python
```

Comment:
Do you actually need to set it back to Python? Does it break other things if you don't?